src/api/bulk2.ts

```diff
@@ -863,7 +863,21 @@ export class IngestJobV2<S extends Schema> extends EventEmitter {
    *
    * @returns Promise<IngestJobV2SuccessfulResults>
    */
-  async getSuccessfulResults(): Promise<IngestJobV2SuccessfulResults<S>> {
+  async getSuccessfulResults(raw?: false): Promise<IngestJobV2SuccessfulResults<S>>
+  async getSuccessfulResults(raw: true): Promise<string>
+  async getSuccessfulResults(raw?: boolean): Promise<IngestJobV2SuccessfulResults<S> | string> {
+    const reqOpts: BulkRequest = {
+      method: 'GET',
+      path: `/${this.id}/successfulResults`,
+    }
+
+    if (raw) {
+      return this.createIngestRequest<string>({
+        ...reqOpts,
+        responseType: 'text/plain',
```

Comment:
jsforce will check `responseType` ( or fallback to `content-type` header ) to try parse the body request to a JS value, passing `text/plain` so it skip parsing.